src/tests/functional/plugin/shared/src/single_layer_tests/op_impl_check/single_op_graph.cpp

```diff
@@ -1409,19 +1409,28 @@ std::shared_ptr<ov::Model> generateMultiSubGraph(const std::shared_ptr<ov::op::O
     }
 }
 
-std::shared_ptr<ov::Model> generateNmsBase(const std::shared_ptr<ov::op::Op> &node) {
+std::shared_ptr<ov::Model> generate(const std::shared_ptr<ov::op::v8::MatrixNms> &node) {
```

Comment:
still v8::MatrixNms?  As other plugins have not support any version MatrixNms, there is no dependency of v8 from other plugins. We can safely update all cases with v9. what do u think?

Reply:
MatrixNms is still v8. Only MulticlassNms upgraded to v9. Not understand this change request here. Can you please give more inputs?

Reply:
Previously MatrixNms and MulticlassNms inherited from NmsBase as they can share validate_and_infer_types(). But now they cannot. So NmsBase is removed.